docs/HailExpressionLanguage.md

```diff
@@ -50,8 +50,10 @@ Several Hail commands provide the ability to perform a broad array of computatio
     - toLong: `str.toLong`
     - toFloat: `str.toFloat`
 
- - Boolean coin flip:
-     - pcoin(p) -- returns `true` with probability `p`. `p` should be between 0.0 and 1.0.
+ - Random Booleans and doubles:
+     - pcoin(p) -- returns `true` with probability `p`. `p` should be between 0.0 and 1.0
+     - runif(max) -- returns a random draw from a uniform distribution on the interval \[0, max\]. `p` should be greater than 0.0.
```

Comment:
`math.random` return something in `[0, 1)`, so it should say `[0, max)`.  The second sentence is wrong.  Also, if you do `runif(-5)` you get something in `(max, 0]`.  Finally, R's `runif(min, max)` returns something in `[min, max]`:

http://stat.ethz.ch/R-manual/R-devel/library/stats/html/Uniform.html

although note this remark, which I don't fully understand:

 > runif will not generate either of the extreme values unless max = min or max-min is small compared to min, and in particular not for the default arguments.